config.analysis

```diff
@@ -41,6 +48,9 @@ climo_yr1 = 6
 climo_yr2 = 10
 # the offset year to be added to simulation years
 yr_offset = 1849
+# start and end date for the analysis
+analysis_start_date = 0001-01-01_00:00:00
+analysis_end_date = 9999-01-01_00:00:00
```

Comment:
Here is perhaps a better option: 1) leave timeseries_start_date and timeseries_end_date in, 2) use *only* the years to select the files, and 3) use the full dates to let xarray to bracket the exact time period. This way we avoid counting too much on filename time stamps, but also have the option to choose specific time periods.